test/ft/actions_defer.cpp

```diff
@@ -5,9 +5,14 @@
 // (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 //
+#include <common/test.hpp>
```

Comment:
Could we instead extend cxx flags to avoid repetition in each test?

```cmake
    # gcc/clang
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -include common/test.hpp)

    #msvc
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /FI common/test.hpp)
```

Reply:
Argh, I didn't found the  MSVC "/FI", so I have added a main to source files :(
and removed "-include" from Makefile
Wasted time ...